src/dep/triemap/triemap.c

```diff
@@ -37,7 +37,7 @@ TrieMapNode *__trieMapNode_resizeChildren(TrieMapNode *n, int offset) {
 /* Create a new trie node. str is a string to be copied into the node,
  * starting from offset up until len. numChildren is the initial number of
  * allocated child nodes */
-TrieMapNode *__newTrieMapNode(char *str, tm_len_t offset, tm_len_t len, tm_len_t numChildren,
+TrieMapNode *__newTrieMapNode(const char *str, tm_len_t offset, tm_len_t len, tm_len_t numChildren,
```

Comment:
all this is meaningless - this is a git subtree and you didn't change the origin of it, it will just get deleted.